doc/api/magic-plot.md

```diff
@@ -126,37 +126,36 @@ Shortcut: `%sqlplot hist`
 
 +++
 
-Histogram does not support NULL values, so let's remove them:
+Histogram supports NULL values, by skipping them:
 
 ```{code-cell} ipython3
-%%sql --save no_nulls --no-execute
+%%sql --save with_nulls --no-execute
```

Comment:
what happens if data contains nulls but user doesn't mention `with_nulls`? will there be an error message?
please add corresponding test case also

Reply:
Also, are we deprecating `no_nulls`? There should be a warning message

Reply:
Here, with_nulls (previously no_nulls) is not a magic-cmd argument, it was just the name by which the snippet was saved to be used later. We are not deprecating `no-nulls` argument. `Histogram` cmd will work if null data present and no additional arguments are needed to enable that.
![Screenshot 2023-06-02 at 11 12 18 AM](https://github.com/ploomber/jupysql/assets/69951190/773d51ac-6bee-46d3-85b8-4e1f38ef1bf9)

I'll make the changes to magic-plot.md example so that snippet-name confusion is resolved.